koku/masu/processor/parquet/parquet_report_processor.py

```diff
@@ -475,7 +485,12 @@ def convert_csv_to_parquet(self, csv_filename):  # noqa: C901
             col_names = pd.read_csv(csv_filename, nrows=0, **kwargs).columns
             if self.ingress_reports:
                 REQUIRED_COLS = set(CSV_REQUIRED_COLUMNS.get(self._provider_type))
+                missing_cols = False
                 if not set(col_names).issuperset(REQUIRED_COLS):
+                    missing_cols = True
+                    if missing_cols and self.col_checker:
```

Comment:
```suggestion
                    if self.col_checker:
```
Total nitpick, but `missing_cols` will never be False here.

Reply:
Whoops! :smile: